openstack/neutron/templates/etc/_neutron-policy.json.tpl

```diff
@@ -332,7 +332,14 @@
     "update_bgpvpn_router_association": "rule:context_is_editor or rule:shared_bgpvpns",
     "delete_bgpvpn_router_association": "rule:context_is_editor or rule:shared_bgpvpns",
     "get_bgpvpn_router_association": "rule:context_is_viewer or rule:shared_bgpvpns",
-    "get_bgpvpn_router_association:tenant_id": "rule:context_is_viewer or rule:shared_bgpvpns"
-
-
+    "get_bgpvpn_router_association:tenant_id": "rule:context_is_viewer or rule:shared_bgpvpns",
```

Comment:
If .Values.interconnection.enabled is disabled, this json list would end with a trailing comma, which is not supported by json and would probably fail to be parsed.

Reply:
Good catch, will fix.